app/decorators/repository_decorator.rb

```diff
@@ -1,19 +1,37 @@
 # frozen_string_literal: true
 
 class RepositoryDecorator < ApplicationDecorator
+  include ActionView::Helpers::NumberHelper
+
   delegate_all
 
   def self.collection_decorator_class
     PaginatingDecorator
   end
 
   def languages
-    return '' unless language
+    return [] unless language
 
-    language.split(',').first(3).to_sentence
+    language.split(', ').first(3).map(&:strip)
   end
 
   def name
     link[/[\w_-]+$/]
   end
+
+  def issues_formatted
+    format_number(model.issues)
+  end
+
+  def stars_formatted
+    format_number(model.stars)
+  end
+
+  private
+
+  def format_number(number)
+    return number.to_s if number <= 999
```

Comment:
why this? you are already formatting the number below

Reply:
I'm doing this, 'cause I want to preserve the real count value from values bellow `999`, instead of use some kind of unit.

Reply:
what it is the return when you remove this and have a number less than 1k ?

Reply:
In the specific case of this test:
https://github.com/Codeminer42/Punchclock/blob/226d7faa9deb7843fb34b0ea00546941dd37f3ee/spec/decorators/repository_decorator_spec.rb#L39

If I remove that condition `number_to_human` method will return `1K` instead of `999`

Reply:
I see no problem in this case, but ok